target/linux/ramips/dts/mt7621_ruijie_rg-ew1200g-pro.dts

```diff
@@ -0,0 +1,142 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "ruijie,rg-ew1200g-pro", "mediatek,mt7621-soc";
+	model = "Ruijie RG-EW1200G PRO";
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power_blue: power_blue {
+			function = LED_FUNCTION_POWER;
+			color = <LED_COLOR_ID_BLUE>;
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	aliases {
+		led-boot = &led_power_blue;
+		led-failsafe = &led_power_blue;
+		led-running = &led_power_blue;
+		led-upgrade = &led_power_blue;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,57600n8";
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&gpio 18 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+};
+
+&spi0 {
+	status = "okay";
+
+	flash@0 {
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		#address-cells = <1>;
+		#size-cells = <1>;
```

Comment:
`#address-cells` and `#size-cells` should be removed as `partitions` node doesn't have `reg` property.

Reply:
Applying that patch causes MTD partitions not to mount during boot with folowing error:

`spi0.0: error parsing ofpart partition /palmbus@1e000000/spi@b00/flash@0/partitions/partition@0 (/palmbus@1e000000/spi@b00/flash@0)`

Reply:
These two lines need to be moved to `partitions` child node.

Reply:
Now there is:

Reply:
In fact, when I started porting, these lines were in `partitions` child note, and "bad cell count" error was fixed by moving these lines to top.